QuantConnect.TradierBrokerage/TradierBrokerage.cs

```diff
@@ -1285,9 +1248,9 @@ private void ProcessPotentiallyUpdatedOrder(TradierCachedOpenOrder cachedOrder,
             if (updatedOrder.RemainingQuantity != cachedOrder.Order.RemainingQuantity
              || ConvertStatus(updatedOrder.Status) != ConvertStatus(cachedOrder.Order.Status))
             {
+                var leanOrderStatus = ConvertStatus(updatedOrder.Status);
                 // get original QC order by brokerage id
-                Order qcOrder;
-                if (!_zeroCrossingOrdersByTradierClosingOrderId.TryGetValue(updatedOrder.Id, out qcOrder))
+                if (!TryGetOrRemoveCrossZeroOrder(updatedOrder.Id.ToStringInvariant(), leanOrderStatus == OrderStatus.Filled, out var qcOrder))
```

Comment:
Move `leanOrderStatus == OrderStatus.Filled,` internally, so it's easier on clients/consumers